### PR TITLE
config: ignore null / absent configuration for the timers (SC-369)

### PR DESCRIPTION
**Pull request by renanrodrigo**
If there is no config entry in `ua_config` for a timer or if it is set to null, just use the default values.

## Desired commit type::
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:
 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Review by blackboxsw — CHANGES_REQUESTED** [review]

File: features/steps/steps.py [hunk 1]
```diff
@@ -434,6 +434,13 @@ def then_stderr_matches_regexp(context):
     assert_that(context.process.stderr.strip(), matches_regexp(context.text))
 
 
+@then("stderr does not match regexp")
+def then_stderr_does_not_match_regexp(context):
+    assert_that(
+        context.process.stderr.strip(), not_(matches_regexp(context.text))
+    )
+
+
```
[the remark this answers is not in the document]
Comment (renanrodrigo) [reply features/steps/steps.py:443]: Thanks - good idea.
Will change it, and update other integration tests as well

[the remark this answers is not in the document]
Comment (blackboxsw) [reply features/steps/steps.py:443]: +1. I don't think other integration tests should need updating then. It should "just work" for existing "stdout does not match regexp"

[the remark this answers is not in the document]
Comment (renanrodrigo) [reply features/steps/steps.py]: @blackboxsw This change was already on main (thanks to you I see (: ), so I just rebased. For consistency, I changed the "matches" step as well.

File: features/steps/steps.py [hunk 2]
```diff
@@ -428,9 +423,10 @@ def then_stream_does_not_match_regexp(context, stream):
     assert_that(content, not_(matches_regexp(context.text)))
 
 
-@then("stderr matches regexp")
-def then_stderr_matches_regexp(context):
-    assert_that(context.process.stderr.strip(), matches_regexp(context.text))
+@then("{stream} matches regexp")
+def then_stream_matches_regexp(context, stream):
```
Comment (blackboxsw) [inline features/steps/steps.py:427]: +1 had wanted to fix that as well in an earler commit/review comment. Thanks for taking it to the next step.

**Review by blackboxsw — APPROVED** [review]

File: lib/timer.py [hunk 3]
```diff
@@ -56,7 +56,15 @@ def run(self, cfg: UAConfig):
...
         """Return the run_interval for the job based on config or defaults."""
         configured_interval = getattr(cfg, "{}_timer".format(self.name), None)
-        if not isinstance(configured_interval, int) or configured_interval < 0:
+        if configured_interval is None:
+            debug_msg = (
+                "No config set for {}, default value will be used."
+            ).format(self.name)
+            LOG.debug(debug_msg)
+            return self._default_interval_seconds
+        elif (
```
[the remark this answers is not in the document]
Comment (renanrodrigo) [reply lib/timer.py:65]: waah I added the return last :|
Thanks for merging though (:

**Pull request merged**